missingno/missingno.py

```
@@ -66,67 +79,73 @@ def matrix(
         ax0.set_yticks(ts_list)
         ax0.set_yticklabels(ts_ticks, fontsize=int(fontsize / 16 * 20), rotation=0)
     else:
         ax0.set_yticks([0, df.shape[0] - 1])
-        ax0.set_yticklabels([1, df.shape[0]], fontsize=int(fontsize / 16 * 20), rotation=0)
+        ax0.set_yticklabels(
+            [1, df.shape[0]], fontsize=int(fontsize / 16 * 20), rotation=0
+        )
 
     # Create the inter-column vertical grid.
     in_between_point = [x + 0.5 for x in range(0, width - 1)]
     for in_between_point in in_between_point:
```

Pylint (3.3.7)

[error] Undefined variable 'in_between_point'

(E0602)